diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 0a7a36bb93eda7a8c5f67e823cf4c76b8f7e2d0d..de5edf155663e5ec25c189336deb6b7dcfa40101 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -166,6 +166,7 @@ exports.networks = [
     { chainId: 3338n, shortName: 'PEAQ' },
     { chainId: 3501n, shortName: 'JFIN' },
     { chainId: 3636n, shortName: 'BTNX' },
+    { chainId: 3637n, shortName: 'BTCm' },
     { chainId: 3737n, shortName: 'csb' },
     { chainId: 3776n, shortName: 'astrzk' },
     { chainId: 4002n, shortName: 'tftm' },
