diff --git a/dist/src/utils/eip-3770/config.js b/dist/src/utils/eip-3770/config.js
index 4937a8edcfb3981d3974d2a0fa9bafdb92cde131..2076c454816289704549fa538afbe74ff417e25c 100644
--- a/dist/src/utils/eip-3770/config.js
+++ b/dist/src/utils/eip-3770/config.js
@@ -24,6 +24,7 @@ exports.networks = [
     { chainId: 42n, shortName: 'kov' },
     { chainId: 43n, shortName: 'pangolin' },
     { chainId: 44n, shortName: 'crab' },
+    { chainId: 3636n, shortName: 'BTNX' },
     { chainId: 46n, shortName: 'darwinia' },
     { chainId: 50n, shortName: 'xdc' },
     { chainId: 51n, shortName: 'txdc' },
